pkg/pipeline_events/elasticsearch.go

```diff
@@ -90,7 +90,10 @@ func (e ElasticsearchProvider) SendRelease(r *v1.Release) error {
 				r.Namespace: r.CreationTimestamp.String(),
 			},
 		}
-		e.SendIssue(&esissue)
+		err = e.SendIssue(&esissue)
```

Comment:
hmmm, did not even know this existed. The file/functionality itself. I am almost certain it can safely be deleted. @abayer? 

Just wrapping the error works for me as well.